### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by tstapleton**
Some feedback addressed from #1 ( @sqren ), and setup for continued work tomorrow.

### CODE REVIEW

File: src/public/js/tradeshift-api.js [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+			xhr.setRequestHeader('Content-Type', 'application/json');
+			xhr.setRequestHeader('Accept', 'application/json');
+			xhr.onload = function() {
+				if (xhr.status === 200) {
+					resolve(xhr.responseText ? JSON.parse(xhr.responseText) : null);
+				} else {
+					reject(xhr.status);
+				}
+			};
+			xhr.send(JSON.stringify(body));
```
Comment (sqren) [inline src/public/js/tradeshift-api.js]: I can recommend [superagent](https://github.com/visionmedia/superagent) for http request. Works on server + client.
(just a heads up - no code change required)

File: src/public/js/tradeshift-api.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+		});
+	}
+
+
 	function getAccountInfo() {
 		var xhr = new XMLHttpRequest();
-		xhr.open('GET', encodeURI('https://a272fd4f.ngrok.io/ts/info'));
+		xhr.open('GET', encodeURI('https://a272fd4f.ngrok.io/ts/info'), true);
```
Comment (sqren) [inline src/public/js/tradeshift-api.js]: What is this? A proxy?

File: src/public/js/tradeshift-api.js [hunk 3]
```diff
@@ -1,9 +1,29 @@
+/*global Q*/
 (function() {
 	var resultArea = document.getElementById('js-api-results');
 
+	function _request(method, path, body) {
+		var host = 'http://localhost:1337/';
```
Comment (sqren) [inline src/public/js/tradeshift-api.js]: Should this come from a proxy instead of being hard-coded?

File: src/views/partials/scripts.hbs [hunk 4]
```diff
@@ -1,2 +1,6 @@
+<script src="js/lib/q.min.js"></script>
+<script src="js/lib/chart.min.js"></script>
+<script src="js/recent.js"></script>
+<script src="js/engage.js"></script>
```
Comment (sqren) [inline src/views/partials/scripts.hbs]: Is browserify/requirejs overkill?

**Comment by sqren** [comment]
Nice job. Looking forward to a full-blown third party nodejs sdk :)
:100:

**Comment by tstapleton** [comment]
Thanks @sqren ! I'll catch the feedback in the next PR.

File: src/public/js/tradeshift-api.js (hunk 1 above)
Comment (tstapleton) [inline src/public/js/tradeshift-api.js]: thanks!

File: src/public/js/tradeshift-api.js (hunk 2 above)
Comment (tstapleton) [inline src/public/js/tradeshift-api.js]: Secure tunnel to localhost so the app can be deployed to a sandbox while running locally - https://ngrok.com/

File: src/public/js/tradeshift-api.js (hunk 3 above)
Comment (tstapleton) [inline src/public/js/tradeshift-api.js]: yep. sort of fixed in #3 .

File: src/views/partials/scripts.hbs (hunk 4 above)
Comment (tstapleton) [inline src/views/partials/scripts.hbs]: I thought it was at the beginning, but now...

Comment (sqren) [inline src/views/partials/scripts.hbs]: If you go down that route, you could use Grunt + browserify as we do normally, or use Webpack. It's a pretty need build tool.

**Pull request merged**